### ISSUE TITLE
Surface water mass transformation

### ISSUE THREAD

**Issue by adele157** [adele157/easterlies-collaborative-project#5]
To do:

- Calculate surface water mass transformation in the 4 dense water formation regions.

- Plot density vs formation rates for control and perturbation simulations averaged over years 11-15.

- Plot time series of dense water formation for given density values.

**Comment by StephenGriffies** [comment]
if you are game to help test some new surface WMT python code, then please let me know as we (at Princeton) are trying to develop community used/supported software.  The point people are Jan-Erik Tesdal and Graeme MacGilchrist, two post-docs at Princeton.  Jan-Erik has already done analysis with some of Ruth's ACCESS output so I suspect the code would work out of the box.  

I realize you already have your own code, so it would in the nearterm not be the most efficient use of your time. My motivation, however, is to ensure that the community surface WMT code we are developing will be of use to ACCESS users, and your input would be very welcome.

**Comment by StephenGriffies** [comment]
Jan-Erik has gone through your notebook and confirms that his are consistent with yours.  However, he makes use of Xarray and some routines from Julius Buseke for binning, each of which speed up the calculation a lot. So if you find efficiency to be a problem with your notebook, then I can point you to Jan-Erik's scripts.

**Comment by adele157** [comment]
Yes, that would be great, thanks!

**Comment by StephenGriffies** [comment]
You are now added to the project.  Thanks

**Comment by adele157** [comment]
Yes, sorry, haven't uploaded here yet. There's one here I've been using for the simulations:
/g/data/v45/akm157/jupyter_scripts/easterlies/SWMT_process_data_easterlies.ipynb

Let me know if you want me to talk you through it, can zoom today.
